### PR TITLE
media-sound/mpd: support systemd in logrotate config

### PR DESCRIPTION
**Pull request by Tatsh**
This enables systemd support in the logrotate `postrotate` script. This is done with `try-restart` as this will only restart mpd if it is running (which is similar to the OpenRC script). The systemd service file provided by mpd does not support the `reload` action.

### CODE REVIEW

File: media-sound/mpd/files/mpd-0.20.21.logrotate [hunk 1]
```diff
@@ -4,6 +4,11 @@
 	delaycompress
```
Comment (Zlogene) [inline media-sound/mpd/files/mpd-0.20.21.logrotate]: Please add this only to the latest version, also please change a revision of the ebuild so users can get the updated during explicit uodate request.

Also `--quiet` option has to remain in the else branch, why did you change this?

And I would rewrite this like:

```
	postrotate
		if type -p systemctl 2>&1 1>/dev/null && systemctl is-system-running 2>&1 1>/dev/null ; then
			systemctl kill -s HUP mpd.service
		else
			/etc/init.d/mpd --quiet reload
		fi
	endscript
```

Comment (Tatsh) [reply media-sound/mpd/files/mpd-0.20.21.logrotate]: Forgot to keep `--quiet` since the old versions that use init don't have that. I have made the changes requested.

**Comment by Zlogene** [comment]
Actuslly I have no idea why add it, systemd rotates logs perfectly fine by itself.
